HW04/ekaterina_balabanovich/lib/work_demonstration.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require_relative 'student'
+require_relative 'mentor'
+require_relative 'homework'
+require_relative 'notification'
+
+class WorkDemonstration
+  def run
+    mentor_one = create_mentor('Ivan', 'Stepanov')
+    mentor_two = create_mentor('Trevor', 'Dutch')
```

Comment:
Minor. We usually use numbers for such variables
```suggestion
    mentor1 = create_mentor('Ivan', 'Stepanov')
    mentor2 = create_mentor('Trevor', 'Dutch')
```